auth0_flutter/lib/src/web/auth0_flutter_plugin_real.dart

```diff
@@ -1,12 +1,15 @@
 import 'dart:async';
 import 'dart:html';
-import 'dart:js_util';
+
 import 'package:auth0_flutter_platform_interface/auth0_flutter_platform_interface.dart';
 import 'package:flutter_web_plugins/flutter_web_plugins.dart';
+
 import 'auth0_flutter_web_platform_proxy.dart';
 import 'extensions/client_options_extensions.dart';
 import 'extensions/credentials_extension.dart';
-import 'js_interop.dart';
+import 'extensions/logout_options.extension.dart';
+import 'js_interop.dart' as interop;
```

Comment:
As we now have `LogoutOptions` in both the platform interface and JS interop, I prefixed the interop types. Kind of useful to distinguish them anyway.